docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup-automatic.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Set up transactional xCluster
+headerTitle: Set up transactional xCluster
+linkTitle: Setup
+description: Setting up transactional (active-active single-master) replication between two YB universes
+headContent: Set up transactional xCluster replication
+menu:
+  preview:
+    parent: async-replication-transactional
+    identifier: async-transactional-setup-1-automatic
+    weight: 10
+tags:
+  other: ysql
+type: docs
+---
+
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../async-transactional-setup-automatic/" class="nav-link active">
+      Automatic
+    </a>
+  </li>
+  <li >
+    <a href="../async-transactional-setup-semi-automatic/" class="nav-link">
+      Semi-Automatic
+    </a>
+  </li>
+  <li >
+    <a href="../async-transactional-setup-manual/" class="nav-link">
+      Manual
+    </a>
+  </li>
+</ul>
+
+{{< note title="Note" >}}
+To use automatic transactional xCluster replication, both the Primary and Standby universes must be running v2.25.1 or later.
+{{< /note >}}
+
+
+Automatic transactional xCluster replication {{<tags/feature/tp>}} handles all aspects of replication for both data and schema changes.
+
+In particular, DDL changes made to the Primary universe are automatically replicated to the Standby universe.
+
+In this mode, xCluster replication operates at the YSQL database granularity. This means you only run xCluster management operations when adding and removing databases from replication, and not when tables in the databases are created or dropped.
+
+
+## Set up Automatic mode replication
+
```

Comment:
I don't think that backup/restore always restores all global objects(?), so I think it would be good to add in an extra pre-step here to double check that global objects match on both sides first

Reply:
Done. Also put a tip to point to that page from this page.
users, roles, tablespaces. @hulien22 let me know if anything else needs to be included.